frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -162,16 +178,28 @@ public void initChannel(Channel ch) {
                                     future -> {
                                         // TODO:
                                         // use gpu, batch size in load model command
+                                        latch.countDown();
                                         RequestBatch input = new RequestBatch();
                                         if (gpuId > 0) {
                                             input.addModelInput(
                                                     new ModelInputs("gpu", String.valueOf(gpuId)));
                                         }
 
                                         Job job =
-                                                new Job(null, model.getModelName(), "load", input);
-                                        model.addFirst(job);
+                                                new Job(
+                                                        null,
+                                                        model.getModelName(),
+                                                        CommandStrings.LOAD,
+                                                        input);
+                                        model.addJob(getName(), job);
+                                        latch.countDown();
```

Comment:
Why countDown twice?

Reply:
I had removed this. I think its updated in the other PR. I will remove it from this PR as well.